GVRf/IO/STL/src/org/gearvrf/stl/STLConstants.java

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
```

Comment:
As far as naming goes I am fine changing STL to something less ambiguous - along the lines of IOSupportLibrary.